templates/localvolumeprovisioner.yaml

```diff
@@ -13,8 +13,6 @@ spec:
   kubernetes:
     minSupportedVersion: v1.14.0
   cloudProvider:
-    - name: aws
```

Comment:
We could use the localvolumeprovisioner on AWS but it should not be the enabled option by default

Reply:
the problem here is that it would default the storage class to true? right?

Reply:
yes, i might be confused with this property (meaning enabled == it can be used but not enabled as the default one).

Reply:
Conclusion: let's keep it false and let the user to enable it themselves